tests/v2/validation/rbac/rbac_additional_test.go

```diff
@@ -184,119 +189,51 @@ func (rb *RBACAdditionalTestSuite) ValidateCannotAddMPMsAsProjectOwner() {
 	assert.Equal(rb.T(), 0, len(userGetProject.Data))
 }
 
-func (rb *RBACAdditionalTestSuite) ValidateListGlobalSettings() {
-	adminListSettings, err := listGlobalSettings(rb.steveAdminClient)
+func (rb *RBACAdditionalTestSuite) TestRBACP1RestrictedAdmin() {
```

Comment:
since this is creating a cluster as part of the test, can you maybe add that in the name of the test?

Reply:
we are also adding other tests in this test but not specifically creating a cluster. let me know if changing it to TestRBACP1RestrictedAdminChecks() makes more sense?

Reply:
2 questions:
* Does the order of the tests in this function matter? if not, can this be broken out into multiple tests
* We should have some way of signifying that some tests outside of provisioning are creating clusters. I can't remember exactly what was discussed, but having a cluster create call in one test, where the others in the file don't seems like a mixed bag. And was hoping to resolve that at least a little by having it in the name? 
  * Or wouldn't this be a case for k3d?

Reply:
hmm. I thought about it and I think it might be better if we split these tests into multiple files for each of these roles. so that there wont be multiple P1 tests with different roles. If we want to add to these tests going forward it would be easier too. 

That being said, to answer your questions, 

- The order matters for some of these tests but not all. 
- We wanted to have these tests in rbac folder because we dont want to have rbac tests scattered across multiple packages given its role based. I'm creating a new folder for these tests and split based on their role - only for this additional tests and not P0 tests. we would be updating this create cluster to use a k3d as well.